src/components/skeletons/SkeletonElement/SkeletonElement.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+
+import Shimmer from '../Shimmer'
+
+import './skeleton-element.scss'
+
+type ElementVariant = 'text' | 'button' | 'image' | 'badge'
+
+interface Props {
+  loading?: boolean
+  shimmer?: boolean
+  type: ElementVariant
+  children?: JSX.Element
+}
+
+function SkeletonElement({
+  type,
+  children,
+  loading = true,
```

Comment:
do we need to pass the loading state in here? Isn't it better to do this if outside of the component?

Reply:
I left that way for the cases without any `children` components, so then it'll keep rendering the skeleton